src/data/list/perm.lean

```diff
@@ -216,6 +216,17 @@ begin
           ⟨r₁, pr.trans pm, sr⟩ }
 end
 
+theorem perm.sizeof_eq_sizeof [has_sizeof α] {l₁ l₂ : list α} (h : l₁ ~ l₂) :
+  l₁.sizeof = l₂.sizeof :=
+begin
+  induction h with hd l₁ l₂ h₁₂ h_sz₁₂ a b l l₁ l₂ l₃ h₁₂ h₂₃ h_sz₁₂ h_sz₂₃,
```

Comment:
No need to change it, but for reference, you don't have to give all these names explicitly since you only use the last two. The others can be `_`. Saves some time writing subscripts.

Reply:
I know, just like the local context to look good. If I were to skip the names, I could also just use the generated names.

Reply:
You can use `case` if you want to improve presentation of inductive proofs like this.